website/templates/log_templates.mako

```diff
@@ -189,8 +182,16 @@ to
 </script>
 
 <script type="text/html" id="project_registered">
-registered
-<a class="log-node-title-link overflow" data-bind="attr: {href: nodeUrl}, text: nodeTitle"></a>
+   {{#if params.is_prereg}}
```

Comment:
This syntax will no longer supported, since we're removing knockout punches. Please use regular Knockout syntax instead.